.github/workflows/ci.yaml

```diff
@@ -12,16 +12,28 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+
       - name: Use Node.js
         uses: actions/setup-node@v3
         with:
           node-version: "18.x"
+
       - name: Setup pnpm
         uses: pnpm/action-setup@v2.2.4
 
       - run: pnpm install
+
       - run: pnpm deduplicate:check
+
       - run: pnpm build
+
       - run: pnpm lint
-      - run: pnpm test
+
+      - name: "Run tests"
+        run: pnpm test
+        env:
+          GATED_SDK_TEST_PK:  ${{ secrets.GATED_SDK_TEST_PK }}
```

Comment:
maybe anyone know some better way to provide env to just `sdk-gated` as this one is polluting our global github actions config

Reply:
you would need to run a separate task only to run tests of sdk-gated and then define the env var there, but I think there is nothing wrong with the way it is now. we just need some env vars naming convention. shall they be prefixed with subpackage name?

Reply:
good point, fixed.